index.html

```diff
@@ -2175,58 +2175,56 @@ <h4 id=el-figcaption>`figcaption`</h4>
     <tr>
       <th>[[wai-aria-1.2]]</th>
       <td>
-        No corresponding role
+        <a class="core-mapping" href="#role-map-caption">`caption`</a> role
       </td>
     </tr>
     <tr>
       <th><a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a></th>
       <td>
         <div class="role">
-          <span class="type">Roles:</span> `ROLE_SYSTEM_TEXT`; `IA2_ROLE_CAPTION`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
         <div class="relations">
-          <span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+          <span class="type">Relations:</span>
+          If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `IA2_RELATION_DETAILS_FOR` with parent `figure`. 
         </div>
       </td>
     </tr>
     <tr>
       <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
       <td>
         <div class="ctrltype">
-          <span class="type">Control Type:</span> `Text`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
       </td>
     </tr>
     <tr>
       <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
       <td>
         <div class="role">
-          <span class="type">Role:</span> `ATK_ROLE_CAPTION`
+          <span class="type">Role:</span> Use WAI-ARIA mapping
         </div>
         <div class="relations">
           <span class="type">Relations:</span>
-          `ATK_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+          If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `RELATION_DETAILS_FOR` with parent `figure` element.
```

Comment:
I have changed this to what I hope is more understandable, but I'm not sure if I have inadvertently changed the meaning. Please double-check before merging the suggestion.

```suggestion
          If this is the first `figcaption` descendant in the related <a href="#el-figure">`figure`</a> element, then expose `RELATION_DETAILS_FOR` referencing the parent `figure` element.
```